src/cards/ScenarioNode/ScenarioNode.js

```diff
@@ -260,13 +263,15 @@ export const ScenarioNode = ({
   const getAccordionDetails = () => {
     return (
       <AccordionDetails className={classes.scenarioDetailsContainer}>
-        <div className={classes.scenarioDetailsNameLine}>{getScenarioNameAndValidationStatus(false)}</div>
+        <div className={classes.scenarioDetailsNameLine}>{getScenarioNameStatusAndRunType(false)}</div>
         {getDetailedStatus()}
+        <Typography className={classes.cardLabel}>{labels.runTemplate}</Typography>
+        <Typography data-cy="scenario-run-template" className={classes.runTemplate}>
```

Comment:
Can we rename `classes.runTemplate` to `classes.runTemplateName`? We're using `labels.runTemplate` just above, but it doesn't use this "class", so the name is misleading

Reply:
In the style file, the class _runTemplate_ is just above the datasets class which get the same attributes.